internal/k8s/ako_init.go

```diff
@@ -463,6 +463,38 @@ func (c *AviController) FullSyncK8s() error {
 		}
 	}
 
+	// Section for Istio
+	if lib.IsIstioEnabled() {
+		// If Istio is enabled, we need to listen to few of the Istio CRDs.
```

Comment:
I think we can add an info log in this section to denote that istio objects are being processed.

Reply:
Will add it in the next PR.